datafusion/results/single.json

```diff
@@ -1,58 +1,58 @@
 {
     "system": "DataFusion (single)",
```

Comment:
I am thinking to rename this to `DataFusion (parquet)` to match `DuckDB (parquet)`.

Thoughts?

Reply:
done